src/scripts.js

```diff
@@ -344,9 +348,38 @@ function populatePantry() {
       <tr>
         <td>${ingredientData.name}</td>
         <td>${item.amount}</td>
+        <td class="button-column">
+          <button class="round-buttons">+</button>
+          <button class="round-buttons">-</button>
+        </td>
       </tr>`
   })
-  console.log(currentUser.name)
+}
+
+function populateDropdown() {
+  const ingData = ingredientsData.filter(ingredient => {
+    return typeof ingredient.name === 'string'
```

Comment:
We believe there's something wrong in the api (one ingredient is coming back undefined) and this is helping us work around it essentially.